Documentation/guides/DotNet5.md

```diff
@@ -110,14 +122,13 @@ The following instructions can be used for early preview testing.
   <PropertyGroup>
     <TargetFramework>netcoreapp5.0</TargetFramework>
     <RuntimeIdentifier>android.21-arm64</RuntimeIdentifier>
+    <OutputType>Exe</OutputType>
   </PropertyGroup>
 </Project>
 ```
 
-  4) Publish (and optionally install) the project:
+  4) Build (and optionally run) the project:
 
-```
-dotnet publish -t:Install *.csproj
-```
+    dotnet build *.csproj -t:Run
```

Comment:
I think we need to also update  https://github.com/xamarin/xamarin-android/blob/master/src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.targets#L33 for this to work?